libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -454,6 +454,23 @@ public static async Task<BatchFailedEntriesResponse> GetPagedFailedEntriesAsync(
             }
         }
 
+        /// <summary>
+        /// Cancels a batch operation by its id.
+        /// </summary>
+        /// <param name="turnContext"> Turn context. </param>
```

Comment:
Let's unify the criteria with the comments' spacing. I'd like no spaces before and after the comments, but let's choose one and keep it.
```suggestion
        /// <param name="turnContext">The turn context.</param>
```